### PR TITLE
PROD-4041: deploy Velocloud Edge appliances in a GCP project. 

### PR DESCRIPTION
**Pull request by scheuk**

### CODE REVIEW

**Review by adm1nx — APPROVED** [review]

File: main.tf [hunk 1]
```diff
@@ -14,7 +14,259 @@
...
+
+  project_id   = var.project_id
+  network_name = "${each.value}-vpc"
+  routing_mode = "GLOBAL"
+
+  subnets = local.subnets[each.value]
+}
+
+module "inet_firewall_rules" {
+  source       = "terraform-google-modules/network/google//modules/firewall-rules"
```
Comment (vliubko) [inline main.tf]: perhaps makes sense to pin the version?

File: main.tf [hunk 2]
```diff
@@ -14,7 +14,259 @@
...
+    }]
+    deny = []
+    log_config = {
+      metadata = "EXCLUDE_ALL_METADATA"
+    }
+  }]
+}
+
+module "lan_firewall_rules" {
+  source       = "terraform-google-modules/network/google//modules/firewall-rules"
```
Comment (vliubko) [inline main.tf]: perhaps makes sense to pin the version?

File: main.tf [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /**
```
Comment (vliubko) [inline main.tf:1]: this `main.tf` is kinda big
I suggest to split it into more files logically grouped (i.e. `main.tf`, `gcp.tf`, `velocloud.tf`)

File: variables.tf [hunk 4]
```diff
@@ -16,8 +16,62 @@
...
+
+}
+
+variable "velocloud_vco" {
+  description = "Base hostname to the Velocloud Orchestrator instance"
+  type        = string
+}
+
+variable "velocloud_token" {
+  description = "API token for the Velocloud Orchestrator instance"
```
Comment (vliubko) [inline variables.tf:58]: Question: do we have only one (prod) instance of Velocloud running, correct? Nonprod doesn't exist?

File: examples/multi_region/README.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
+# Simple Example
+
+This example illustrates how to use the `sdwan` module.
```
Comment (vliubko) [inline examples/multi_region/README.md:3]: I like the examples included to the module 👍

File: main.tf (hunk 3 above)
Comment (scheuk) [reply main.tf:1]: Makes sense to me, it is a little long and hard to reed. I will attempt to break apart the main.tf file

File: main.tf (hunk 2 above)
Comment (scheuk) [reply main.tf]: yup, will add a version constraint here.

File: main.tf (hunk 1 above)
Comment (scheuk) [reply main.tf]: and here

File: variables.tf (hunk 4 above)
Comment (scheuk) [reply variables.tf:58]: We do not, just a single instance for now. 
We might have a second one, but I'll need to spend some time figuring that out.

File: examples/multi_region/README.md (hunk 5 above)
Comment (scheuk) [reply examples/multi_region/README.md:3]: yes, in fact the example is what gets tested.

**Review by vliubko — APPROVED** [review]

**Pull request merged**